style.css

```diff
@@ -0,0 +1,172 @@
+@import url('base/reset.css');
+:root{
+    --black:#000000;
+    --dark-purple:#4C3FAF;
+    --light-purple:#7C6FE2;
+    --primary-color:#e2e2e2;
+    --secondary-color:#6050dc;
+    --white:#FFFFFF;     
+}
+
+/* Common styles */
+
+.cursor{
```

Comment:
You can place the styles mentioned in a common stylesheet apart from this main one because you might be needing it frequently!!

Reply:
Thankyou for your review, I have separated the common styles in the base folder.